### PR TITLE
Add red dot FAQ

### PR DESCRIPTION
**Pull request by abeatrix**
Just want to clarify because users have mistaken the red dot for connection issues. 

image

### CODE REVIEW

File: doc/admin/how-to/troubleshoot-sg-extension.md [hunk 1]
```diff
@@ -26,6 +26,10 @@ This happens if the extension is disabled or if you're visiting a page where an
 You can upgrade an extension in your private extension registry by simpily running the same `src extensions copy -extension-id=... -current-user=...` command as you would when you first [publish the extension](https://docs.sourcegraph.com/admin/extensions#publish-extensions-to-a-private-extension-registry).
 
 
+#### What does it mean when a red dot shows up on the Sourcegraph browser extension icon?
+
+The red dot indicates that you have enabled Sourcegraph on this specific domain. More infomration can be found in our [browser extension docs](https://docs.sourcegraph.com/integration/browser_extension#make-it-work-on-your-code-host).
```
Comment (tjkandala) [inline doc/admin/how-to/troubleshoot-sg-extension.md:31]:
```suggestion
The red dot indicates that you have enabled Sourcegraph on this specific domain. More information can be found in our [browser extension docs](https://docs.sourcegraph.com/integration/browser_extension#make-it-work-on-your-code-host).
```

**Review by tjkandala — COMMENTED** [review]
We show the red dot [when either the Sourcegraph URL is invalid or the user is on a private repository](https://sourcegraph.com/github.com/sourcegraph/sourcegraph@main/-/blob/client/browser/src/browser-extension/scripts/backgroundPage.main.ts?utm_source=VSCode-1.3.0#L451:5) (we'll remove this once we support private repositories in the browser extension). Is there a case in which we show the alert but the user is able to use the extension without problems?

**Comment by abeatrix** [comment]
> We show the red dot [when either the Sourcegraph URL is invalid or the user is on a private repository](https://sourcegraph.com/github.com/sourcegraph/sourcegraph@main/-/blob/client/browser/src/browser-extension/scripts/backgroundPage.main.ts?utm_source=VSCode-1.3.0#L451:5) (we'll remove this once we support private repositories in the browser extension). Is there a case in which we show the alert but the user is able to use the extension without problems?

Hey @tjkandala sorry I just saw this. Thanks for clarifying on the red dot alert, I happened to locate the same code yesterday but here is an example where I am able to use the extension with the alert on the icon: https://bitbucket.sgdev.org/projects/PRIVATE/repos/test-repo/browse/sum.js
image

example 2: https://bitbucket.sgdev.org/projects/SOURCEGRAPH/repos/jsonrpc2/browse/conn_opt.go
image


Alert sometimes doesn't show up when I'm in a private repo but nothing would work:
image

**Review by tjkandala — APPROVED** [review]
LGTM

**Pull request merged**